### DISCUSSION TITLE
Plugin locks me out of VeSync account

### DISCUSSION THREAD

**Discussion by snbronson** [RaresAil/homebridge-levoit-air-purifier#104] [Q&A]
Am I the only person having this issue?

This plugin seems to be making excessive requests to the VeSync server, resulting in them locking my account. When the plugin is active in Homebrige, I am able to see my Air Purifiers in the Home app and in the VeSync app. At some point, however, I lose access in both places.

I contacted their support and was told the following:

“For a little while the company's business requirements, which have necessitated restrictions on APP requests to prevent malicious activity. Exceeding this limit will result in the APP becoming unavailable until the following day.
 
Your frequent use of the APP or research into product usage may lead to an excessive number of requests. To address this, we advise you to close the APP completely when not in use. Refrain from lingering on the HOME screen after opening the APP, as the device list generates a request every 15 seconds. By avoiding the device list interface, you should not encounter this issue again.
 
We hope this helps and thanks for your understanding. Please feel free to contact us if you have any further questions.”

My issue is not resulting from using their app, it is almost certainly this Homebridge plugin because the problem does not recur when I disable the plugin.

**Comment by RaresAil** [comment]
Hey, how many people are sharing the home in home app and how many devices do you have? As i never encountered this issue

**Reply by snbronson** [reply]
Six people are sharing the home. I assume you're asking about Levoit products... I have 6.

**Reply by RaresAil** [reply]
Hmm, that could be the reason for the issue since for each user, it makes 1 call per device, which means 36 calls if everyone opens the control center (if you have the home status there), need to think on a way to resolve this, maybe adding caching to reduce a bit the number of calls

**Comment by snbronson** [comment]
Ok. I am not sure what changed, but this just started happening very recently. There has been no change in my setup in well over a year, probably longer.

On Sep 16, 2024, at 6:34 AM, Nicolae-Rares Ailincai ***@***.***> wrote:



Hmm, that could be the reason for the issue since for each user, it makes 1 call per device, which means 36 calls if everyone opens the control center (if you have the home status there), need to think on a way to resolve this, maybe adding caching to reduce a bit the number of calls
